src/sdi-notify.c

```diff
@@ -38,7 +38,7 @@ struct _SdiNotify {
 
 G_DEFINE_TYPE(SdiNotify, sdi_notify, G_TYPE_OBJECT)
 
-static gboolean launch_desktop(GApplication *app, const gchar *desktop_file) {
+static bool launch_desktop(GApplication *app, const gchar *desktop_file) {
```

Comment:
To me this looks like the linter used by tiobe isn't aware of glib conventions. But I don't have much experience with glib, so I'd suggest you ask someone more knowledgeable than me to review this :)

Reply:
Yes, that's a good part of the problem: glib predates stdbool, so it uses an `int` as a boolean, FALSE is defined as `0` and TRUE is `!!FALSE`, which makes a "funny" mix. That's why I changed this one, but not others, where the gboolean is a must for some reason.

Reply:
But you are right... it makes no sense to fix something that is fine... I'll revert the gboolean -> bool changes

Reply:
Done.

Reply:
There are a couple of `TRUE`->`true` and `FALSE`->`false` changes, that probably need to be reverted as well.

Reply:
In fact, those two cases are errors: they should be as they are now, because they are gboolean, so they should use TRUE and FALSE.